pages.pt_BR/linux/units.md

```diff
@@ -0,0 +1,24 @@
+# units
+
+> Realiza a conversão entre duas unidades de medida.
+> Digitar 'search{{text}}' no console retornará uma lista de todas as unidades que contêm {{text}}.
```

Comment:
The `'` is a bug in the English page, and you can add the `More information` link:
```suggestion
> Digitar `search{{text}}` no console retornará uma lista de todas as unidades que contêm `{{text}}`.
> Mais informações: <https://www.gnu.org/software/units/units.html>.
```
You can fix the English page in this PR too.